selection/randomized/tests/test_sampling.py

```diff
@@ -95,6 +116,10 @@ def test_optimization_sampler(ndraw=20000, burnin=2000):
 
         W = np.ones(X.shape[1]) * 1
         conv = const(X, Y, W, randomizer=rand)
```

Comment:
Here we could say, add the keyword argument randomizer_scale=0.1 and use it in the randomization_ppf below

Reply:
Oh true so that then we don't need to remember the scale above.

Reply:
I will change this now.